fhirpipe/extract/mapping.py

```diff
@@ -192,6 +214,9 @@ def find_cols_joins_and_scripts(tree):
         # If we are not in a leaf, we recurse
         if "attributes" in tree.keys() and tree["attributes"]:
```

Comment:
Should we remove those 3 lines?

Reply:
No, we shouldn't! In the datamodel, attributes of resources are in a field named "attributes" and children attributes of an attribute are in a field named "children"

Reply:
Ok thanks for clarifying :)